pkg/kubelet/kubelet_pods.go

```diff
@@ -799,8 +800,7 @@ func (kl *Kubelet) HandlePodCleanups() error {
 // another goroutine isn't already in action.
 func (kl *Kubelet) podKiller() {
 	killing := sets.NewString()
-	resultCh := make(chan types.UID)
-	defer close(resultCh)
+	lock := sync.Mutex{}
```

Comment:
nit: add a one-line comment explaining that the mutex is used to guard the mutation of the `killing` set.